### DISCUSSION TITLE
Recommended backend

### DISCUSSION THREAD

**Discussion by r00tr4v3n** [firefly-iii/firefly-iii#8357] [Firefly III]
Hello,
many of my selfhosted apps recommend Mysql for performance and reliability reasons. I did not see any warnings for firefly. Is there a reason I should use an entire database server instead of a simple sqlite file? How would I migrate back to SQlite?

**Comment by JC5** [comment]
Both are fine. You are now on MySQL, yes? Just keep it like it is.

**Reply by r00tr4v3n** [reply]
The MYSQL folder uses 200MB of space instead of a 1MB SQlite file. I'm just thinking about the long term space difference in backups (I'm just backing up docker volumes, don't want to sqldump every app's database in yet another script).

**Reply by JC5** [reply]
I'm afraid I'm not sure how to make the switch.

**Reply by r00tr4v3n** [reply]
Is it the same database scheme?

**Reply by JC5** [reply]
Yes it is :)

**Comment by r00tr4v3n** [comment]
This works great for the migration: [https://pypi.org/project/mysql-to-sqlite3/](https://pypi.org/project/mysql-to-sqlite3/)
Hint: The Link in the [docs](https://docs.firefly-iii.org/references/faq/install/#i-want-to-use-sqlite) has a missing dot. The SQlite path is either ./storage/database/database.sqlite (as stated in the touch code example) or /var/www/html/storage/database/database.sqlite

**Reply by JC5** [reply]
Nice find, I'll fix that!
